tutorial/tutorial.md

```diff
@@ -0,0 +1,245 @@
+SPDX is an open standard for communicating license and copyright information.
+Each license is uniquely identified by a shortform [SPDX
+identifier](https://spdx.org/ids). The SPDX identifier for your chosen license
+is GPL-3.0-or-later.
+
+<!-- TODO: Use tool instead -->
+
+You create a `LICENSES` directory in your project root which will contain all
+the licenses that you use in your project. You then download your license
+from the
+[license-list-data](https://github.com/spdx/license-list-data/tree/master/text)
+repository and put it in the `LICENSES` directory.
+
+## 2. Add copyright and license information to each file
+
+Now that you have a license, you need to indicate in the relevant files that
+these files fall under that license. You edit the comment header of
+`src/main.c` as such:
+
+```
+/*
+ * SPDX-Copyright: 2019 Jane Doe <jane@example.com>
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+```
+
+<!-- FIXME: Fix links -->
+
+The `SPDX-Copyright` tag records the publication years and copyright holder
+of the contents of the file. You can read more about [which publication years
+to use]() and [what copyright holders are]() in the FAQ.
+
+The `SPDX-License-Identifier` tag is followed by a [valid SPDX License
+Expression](https://spdx.org/specifications), typically just the SPDX
+Identifier of the license.
+
+<!-- FIXME: Fix links -->
+
+Each file must always contain these two tags in the header. You are allowed
+to [use the tags multiple times if you have multiple copyright holders or
+licenses]().
+
+In the example project, you also edit `Makefile` and `README.md` using this
+header information, but of course with corresponding comment syntax.
+
+### Binary and uncommentable files
+
+You also want to license your image files under GPL-3.0-or-later.
+Unfortunately, images and other binary files do not have comment headers that
+one can easily edit. Other examples include automatically generated files and
+certain data and configuration files for which comments are non-trivial.
+
+There is a simple trick to circumvent this. Create the files `cat.jpg.license`
+and `dog.jpg.license`, each containing the same information about license and
+copyright holder as above.
+
+<!-- TODO: Add link to DEP-5 possibility. -->
+
+### Change licensing information
+
+You discover that the photo of the cat was not licensed under the GPL at all,
+but under Creative Commons Attribution 4.0 International.
+
+The SPDX Identifier of this license is CC-BY-4.0.  You create
+the file `LICENSES/CC-BY-4.0.txt`, following the same steps you used for
+GPL-3.0-or-later.
+
+You then edit `cat.jpg.license` to say:
+
+```
+SPDX-Copyright: 2017 Joe Somebody <joe@example.com>
+
+SPDX-License-Identifier: CC-BY-4.0
+```
+
+### Build artifacts
+
+When you compile your program, you generate some build artifacts, such as
+`src/main.o`.  You do not need to provide any licensing information for those
```

Comment:
```suggestion
`src/main.o`.  As you likely do not intend to distribute those, you do not need to provide any licensing information for those
```

Reply:
I'm not really sure about the clarification here. For REUSE compliance, you don't need to provide licensing information about build artifacts. We're targeting mostly FOSS developers here, who don't really bother about licensing build artifacts because you can always derive that from the source components. If you want to license build artifacts anyway, then there are better ways to do that than REUSE.

This seems like a FAQ entry to me. "How do I license build artifacts?" Then the complications can be left out of the tutorial.

Reply:
My thinking here (and the comment above) is in REUSE we don’t care about build artefacts _because_ they are not distributed further. If they ended up on the downstream’s disk either through `git clone` or by unpackaging an archive, it would need to be tracked.